lib/assert/suite.rb

```diff
@@ -62,37 +67,33 @@ def count(thing)
       end
     end
 
-    def test_count
-      self.tests.size
-    end
+    # Test Handling
 
-    def result_count(type = nil)
-      if type
-        self.tests.inject(0) do |count, test|
-          count += test.result_count(type)
-        end
-      else
-        self.results.size
-      end
-    end
+    def ordered_tests;                      end
+    def reversed_ordered_tests;             end
+    def ordered_tests_by_run_time;          end
+    def reversed_ordered_tests_by_run_time; end
```

Comment:
@jcredding yeah, I was planning on implementing them in a coming effort when I also rework the view to fully use the new suite api.  This was intended as a bare-minimum to get the existing behavior working.